### PR TITLE
Added Home Page

### PR DESCRIPTION
**Pull request by SimranjitKaurMaan**
Added Home Page
1. Added Banner
2. Added Categories
3. Added Trending Products
image

### CODE REVIEW

File: src/services/APIHandler.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * @param {The name of API to be called to fetch data} apiName 
+ * @returns {response from the API}
+ */
+export const fetchData = async (requestType, apiName) => {
+    const headers = {
+        Accept: 'application/json'
+    }
+    try
+    {
+        const response =  await fetch(apiName, {
```
Comment (rahulcs754) [inline src/services/APIHandler.js:20]: I think here name Should be apiUrl

**Review by rahulcs754 — APPROVED** [review]
The Code looks all good.
you can import these files into Cloudinary and access the URL of the image which will be good.
nice to create separate API call utility files

File: src/services/APIHandler.js (hunk 1 above)
Comment (SimranjitKaurMaan) [reply src/services/APIHandler.js:20]: Thanks will correct this.

**Comment by SimranjitKaurMaan** [comment]
> The Code looks all good. you can import these files into Cloudinary and access the URL of the image which will be good. nice to create separate API call utility files

Sure. I am using cloudinary only. Just forgot to remove assets in project

**Review by raman1999 — APPROVED** [review]
code looks clean to me

**Pull request merged**